tests/models/test_horovod.py

```diff
@@ -376,3 +377,43 @@ def configure_optimizers(self):
 
     # Called every 3 steps, meaning for 1 epoch of 11 batches, it is called 3 times with gamma=0.1
     assert pytest.approx(init_lr * 0.1) == adjusted_lr2
+
+@mock.patch('torch.save')
+@RunIf(special=True, horovod=True, skip_windows=True, min_gpus=2)
```

Comment:
special=True will skip this test.
In order to run it, it has to be added to `tests/special_tests.sh`